emily/handler/src/api/handlers/deposit.rs

```diff
@@ -509,10 +509,13 @@ async fn update_deposits(
     // Loop through all updates and execute.
     for (index, update) in validated_request.deposits {
         if update.is_err() {
+            // We just checked that update is err, so this unwrap is fine.
+            let error = update.unwrap_err().to_string();
```

Comment:
I would change this to be something like
```suggestion
        if let Err(error) = update {
```
These lines could also benefit from some more `match` statements, but no need to make that change.

Reply:
[58310c7](https://github.com/stacks-sbtc/sbtc/pull/1743/commits/58310c7d823f1fea102c96120a0fbde811e8b772)